### PR TITLE
Add My Profile

### PR DESCRIPTION
**Pull request by chiraglal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1716"></a>

### CODE REVIEW

**Review by vedanshi555 — CHANGES_REQUESTED** [review]
why have you added your github profile link in your portfolio ?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: public/data/Chirag-Lal.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Chirag Lal",
+  "type": "personal",
+  "bio": "Front-End Web Developer with a keen interest towards Movies and Music",
+  "avatar": "https://github.com/chiraglal.png",
+  "links": [
+    {
+      "name": "Check Out my Portfolio",
+      "url": "https://github.com/chiraglal",
```
Comment (Panquesito7) [inline public/data/Chirag-Lal.json:9]: I think you should either keep this field or remove the GitHub one.
I recommend adding your portfolio here if you have one. If you don't have one, IMO, it's best not to add the field. :)

What do you think?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (chiraglal) [reply public/data/Chirag-Lal.json:9]: oh sorry, my bad. I will add my portfolio link.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by vedanshi555 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**